cassandra/protocol.py

```diff
@@ -126,10 +126,13 @@ def __init__(self, code, message, info):
         self.info = info
 
     @classmethod
-    def recv_body(cls, f, protocol_version, *args):
+    def recv_body(cls, f, protocol_version, protocol_features, *args):
         code = read_int(f)
         msg = read_string(f)
-        subcls = error_classes.get(code, cls)
+        if protocol_features != None and code == protocol_features.rate_limit_error:
```

Comment:
Instead of checking `None` here, let's guarantee that `protocol_features` it's never none if that's possible.
Even if the server supports no features, the instance of `ProtocolFeatures` can still be non-`None` without any problems.